README.md

```diff
@@ -55,6 +55,39 @@ You may want to do an initial sync of your database by applying the most recent
 app/manage.py migrate
 ```
 
+## Importing Data from the BOD
+
+The "Betriebsobjekte Datenbank" (BOD) is a central database for running and configuring the map
+viewer and some of its services. It contains metadata and translations on the individual layers
+and configurations for display and serving the data through our services such as Web Map Service
+(WMS), Web Map Tiling Service (WMTS) and our current api (mf-chsdi3/api3).
+
+You can import a BOD dump and migrate its data:
+
+```bash
+make setup-bod
+make import-bod file=dump.sql
+app/manage.py bod_migrate
+```
+
+To generate more BOD models, run:
+
+```bash
+app/manage.py inspectdb --database=bod
+```
+
+Although the BOD models are unmanaged (no migrations applied), we nevertheless need migrations for
+testing. Since these are only used for testing, you might remove the initial migration and recreate
+it:
```

Comment:
Rephrased
> The BOD models are unmanaged, meaning Django does not manage any migrations for these models.
However, migrations are still needed during tests to set up the test BOD. To achieve this, it is
necessary to create migrations for the models and dynamically adjust the `managed` flag based on
whether the tests or the server is running (`django.conf.settings.TESTING`). Since these migrations
are only for testing purposes, the previous migration file can be removed and recreated: